### ISSUE TITLE
Fix the relative positioning on member avatars

### ISSUE THREAD

**Issue by matrixbot** [vector-im/vector-web#436]
Created by @ matthew:matrix.org.

**Comment by Fil** [comment]
As an indication, in bundle.css line 2 the following somehow helps a bit (but I don't think it's the correct way to fix it)

```
.mx_MemberAvatar_initial {
    position: relative;
    left: 17px;
    top: -6px;
```

**Comment by ara4n** [comment]
this was caused by 23d45d7f33bcfe7a06ef2eaaf982ad99ab20706f; mending FF broke Chrome intermittently (yay). Now fixed in af6bd53 hopefully.
